setup.cfg

```diff
@@ -100,11 +100,10 @@ tests =
     pytest-rerunfailures~=12.0
     func-timeout~=4.3
     pytest-cov~=4.1.0
-    pandas>=1.5,<3.0
+    pandas>=2.1.2,<3.0
 
 pandas =
-    pandas>=1.5,<3.0
-    numpy<2.0.0
+    pandas>=2.1.2,<3.0
```

Comment:
Does pandas 1.x no longer work?

Reply:
If newer versions don't support python 3.8, that's fine since we are removing it.

Does 1.5.x support python 3.9 and above?

I would like to keep the ranges as big as possible.

Reply:
1.5.x supports 3.8-3.11

Reply:
With that being the case, should we revert this change then, and continue to support `pandas>=1.5,<3.0`?

pip should handle solving for the available dependencies in this case; we are not excluding anyone who still wants/needs to use v1 of pandas.